pom.xml

```diff
@@ -10,7 +10,7 @@
         <version.junit5>5.10.3</version.junit5>
         <version.okhttp>4.12.0</version.okhttp>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <version.unleash.specification>5.1.5</version.unleash.specification>
+        <version.unleash.specification>5.1.9</version.unleash.specification>
```

Comment:
Does it work well with https://github.com/Unleash/unleash-client-java/pull/260 where you had to roll down?

Reply:
It does not! It's on the list of things to do prior to release - a proper set of tests around Windows + unicode. But this is okay for now so these PRs don't get too big I think